app/Repositories/CacheUserRepository.php

```diff
@@ -208,7 +208,7 @@ protected function resolveUpdatedRoles($id, $role)
     protected function resolveMissingUsers($users)
     {
         foreach ($users as $key => $value) {
-            if ($value === false) {
+            if ($value === false or $value === null) {
```

Comment:
@sbsmith86 just noting here that I created an issue #93 to look into why this is weird and can give different expected results to require checking for both